terasoluna-tourreservation-web/pom.xml

```diff
@@ -46,13 +47,12 @@
         <!-- == Begin TourReservation == -->
         <dependency>
             <groupId>${project.groupId}</groupId>
+            <artifactId>terasoluna-tourreservation-env</artifactId>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
@btyanagawamg 
See [Changes](https://github.com/terasolunaorg/terasoluna-tourreservation/compare/issues/494_apply_multiple_module_project_5.0.x...issues/494_apply_multiple_module_project_5.0.x-confirm?expand=1), and [Result of Travis CI](https://travis-ci.org/terasolunaorg/terasoluna-tourreservation/builds/267724944?utm_source=github_status&utm_medium=notification).
If only change scope in domain, Logback.xml in env does not apply and a lot of logs are output.